File: templates/typescript/samples/sample-assistant/src/services/botSettings.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { BotSettingsBase } from 'bot-solutions';
import { TokenExchangeConfig } from '../tokenExchange/';

export interface IBotSettings extends BotSettingsBase {
    tokenExchangeConfig: TokenExchangeConfig;
}

File: samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Services/BotSettings.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Microsoft.Bot.Solutions;
using VirtualAssistantSample.TokenExchange;

namespace VirtualAssistantSample.Services
{
    public class BotSettings : BotSettingsBase
    {
        public TokenExchangeConfig TokenExchangeConfig { get; set; }
        public bool LogPersonalData { get; set; }
    }
}